TWApp/src/main/java/savageTW/Tweets.java

```diff
@@ -0,0 +1,49 @@
+package savageTW;
+
+import com.google.gson.Gson;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.stream.Collectors;
+
+public class Tweets extends HttpServlet {
+    private static Posts posts = new Posts();
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        response.setContentType("application/json");
+
+        String[] uris = request.getRequestURI().split("/");
+
+        if (uris.length == 3 && uris[2].equals("search")) {
+            Gson gson = new Gson();
+
+            response.getWriter().print(posts.getAll().stream().map(gson::toJson).
```

Comment:
(cosmetic) Please do not break up method chain like this
A method is transfered to a next line stating with .dot

Or rather, you either write().the().whole().chain().in().one().line()
or, you()
.split()
.at()
.every()
.method()

You don't just split in one arbitrary place )